packages/mdc-list/README.md

```diff
@@ -1,49 +1,23 @@
-<!--docs:
```

Comment:
Was this removed by accident? I thought we need it as front matter for publishing this on material.io?

Reply:
Ah yes, you're right. Thanks for catching.